src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -795,6 +814,52 @@ private void CloseColumnResize()
         StateHasChanged();
     }
 
+    private void LoadStateFromQueryString(string queryString)
+    {
```

Comment:
Can you add this test, like you did for the Save method (even if the test is already present in the `OnInitialize`, which remains interesting for performance questions)?

```csharp
if (SaveStateInUrl)
{
    return;
}
```